cd/deploy.sh

```diff
@@ -80,8 +80,9 @@ if [ "$RELEASE" == 'true' ] && [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PU
                 \"draft\": false
             }"
         echo $JSON_DATA
-        curl -v --data "$JSON_DATA" https://api.github.com/repos/gchq/$repoId/releases?access_token=$GITHUB_TOKEN
 
+        curl -v -H "Authorization: token $GITHUB_TOKEN" --data "$JSON_DATA" https://api.github.com/repos/arijitdatta/Gaffer_dummy/releases
```

Comment:
just need to switch the repo name back to https://api.github.com/repos/gchq/$repoId/releases

Reply:
Yikes! what a miss!